### PR TITLE
Ticket 5 get articles

### PR DESCRIPTION
**Pull request by PhilTBatt**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -64,12 +66,64 @@ describe('/api/articles/:article_id', () => {
...
+    it('GET: 200 - should return articles sorted by date in descending order', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.Articles).toBeSortedBy('created_at', { descending: true })
+            })
+    })
+
+    it('GET: 404 - should return an error when no articles are found', () => {
```
Comment (nc-scarlett-adams) [inline __tests__/app.test.js:118]: If there're no articles, you should still get 200 OK as the right data- an empty array comes back from the db. This test is a bit unnecessary

**Review by nc-scarlett-adams — COMMENTED** [review]
Great tests and solution. The last test for sad path was a bit unnecessary. Otherwise all good!

**Comment by PhilTBatt** [comment]
The ticket said to consider what errors could occur here and to test for them. Are there no errors to test for then?

File: __tests__/app.test.js (hunk 1 above)
Comment (PhilTBatt) [reply __tests__/app.test.js:118]: The ticket said to consider what errors could occur here and to test for them. Are there no errors to test for then?

Comment (nc-scarlett-adams) [reply __tests__/app.test.js:118]: yeah, there is no errors to test for this endpoint rather then invalid endpoint. And if you've already had that covered, then it's all good for this endpoint

**Pull request merged**